resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml

```diff
@@ -29,6 +29,8 @@ spec:
                     type: string
                   port: 
                     type: integer
+                  es_index:
```

Comment:
Should we rename this to just index as es is redundant since its in the elasticsearch object?

Reply:
I named it `es_index` since index is the name of a function in Jinja and will crash.

Reply:
I changed it to index_name

Reply:
I eventually changed it for `index_name`